### PR TITLE
Default graph config

### PR DESCRIPTION
**Pull request by jsoppela**
Make sure CLI is always using a default graph config defined in config.py

### CODE REVIEW

**Comment by jonathansberry** [comment]
Why can't we use the default value so that it works if you don't define an override from an env var?

**Comment by jsoppela** [comment]
> It seem a bit strange storing the production graph under test data?

If it makes sense, we can move it to the root directory and keep only the artificial test graphs in test_data

File: navigator_engine/config.py [hunk 1]
```diff
@@ -11,11 +11,12 @@ class Config(object):
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     LOGGING_LEVEL = logging.INFO
     DASH_REROUTE_PREFIX = '/engine'
+    DEFAULT_DECISION_GRAPH = f'{base_directory}/Estimates 22 BDG [Final].xlsx'
 
 
 class Testing(Config):
     TESTING = True
-    SQLALCHEMY_DATABASE_URI = f'sqlite+pysqlite:////{base_directory}/../testing.db'
```
Comment (jonathansberry) [inline navigator_engine/config.py:18]: @jsoppela Why the change here?

Comment (jsoppela) [reply navigator_engine/config.py:18]: This was not an intended change, I have reverted it.
